### ISSUE TITLE
Video playback keeps repeating

### ISSUE THREAD

**Issue by DB628** [deckerst/aves#1359]
I've checked all settings within the app, when I play a random video from my Aves gallery, even though I have everything in the video settings disabled, when the video plays, it keeps repeating the playback. So is there a possibility to turn that feature off, if I missed it somewhere please let me know. 

DB

**Comment by deckerst** [comment]
Have you checked Settings > Video > Playback > Loop mode?

By default it's set to "short videos only", so that gif-like videos loop.

**Comment by DB628** [comment]
![Screenshot_20241226_131546_Aves](https://github.com/user-attachments/assets/8f7da6fa-c72a-4778-961a-8828554d6d63)


Yes as you can see in the attached pic, hopefully picture attached as I'm on mobile and driving

**Comment by deckerst** [comment]
Could you please provide a sample file with which you have the issue?

Also, please provide the system info and logs you get from the app About > Bug Report. Best is to export logs after you experience the issue.

**Comment by Nanobot567** [comment]
Hey there! Just ran into this issue as well. Here are the logs from my device, and a video which triggers the bug.

Thanks!

(Also, sorry for bumping a closed issue, I can create a new one if needed)

https://github.com/user-attachments/assets/d59d4e41-69b0-43af-89ca-3d0566cb3ded
[aves-logs-20260120_163141.txt](https://github.com/user-attachments/files/24752690/aves-logs-20260120_163141.txt)
[avesinfo.txt](https://github.com/user-attachments/files/24752691/avesinfo.txt)

```
Package: deckers.thibault.aves
Installer: com.machiav3lli.fdroid
Aves version: 1.13.12-izzy, build 16302
Flutter: stable 3.27.4
Android version: 16, API 36
Android build: BP2A.250605.031.A3.S931USQS7BYLR
Device: samsung SM-S931U
Display: pixel ratio=3.0, logical=360.0x780.0, physical=1080.0x2340.0
Support: dynamic colors=true, geocoder=true, HDR=true, wide gamut=true
Mobile services: ready
Connectivity: wifi
System locales: en_US
Storage volumes: /storage/emulated/0/
Storage grants: /storage/emulated/0/iPhoneData/, /storage/emulated/0/DCIM/
Error reporting: false
```

**Comment by deckerst** [comment]
@Nanobot567 Have you checked that Settings > Video > Playback > Loop mode is set to `Never`?

**Comment by Nanobot567** [comment]
Yes, here are my current video playback settings -

![Image](https://github.com/user-attachments/assets/a4a7c8e0-a5ee-4f8b-91e6-aded3884f399)

**Comment by deckerst** [comment]
@Nanobot567 could you please try [this build](https://drive.google.com/file/d/1PowfLB2wpryeOf_zlRv1VLn2KjtHqasU/view?usp=sharing)? It will install as a separate app named `Aves [Profile]`, with its own settings, etc.

If you still experience the issue with this build, please export and attach here the logs as you did before.

**Comment by Nanobot567** [comment]
Just gave that build a go, unfortunately the issue's still present :( thanks for the quick turnaround though!

The first time I played the video through to completion it did seem to be fixed, but after playing the video again the issue returned.

I can't give you the logs as it looks like the log file "Save" button is broken in this build 😅

**Comment by deckerst** [comment]
> I can't give you the logs as it looks like the log file "Save" button is broken in this build 😅

Do you mean it yields an empty file?

Otherwise, if you know your way around `logcat` you could save its output filtered for `deckers.thibault.aves.profile`.

**Comment by Nanobot567** [comment]
Whenever I tap on the button no "save file" dialog appears. But that's alright, here's the log from logcat:

[aves-log.log](https://github.com/user-attachments/files/24780577/aves-log.log)

**Comment by deckerst** [comment]
Thanks, that's helpful. At least to confirm that the video player is configured as designed. So I really wonder why it's looping on your device.

By the way, I see some logs referring to `Kumiho-Kumiho`. Is it a custom ROM, or something else?

**Comment by Nanobot567** [comment]
Yeah I noticed that too when I glanced at the logs, honestly I'm not entirely sure what Kumiho-Kumiho is! It might have something to do with Shizuku as I do have that installed (searching for compatible apps maybe?), but otherwise I have no clue. May have to do some research on that, haha.

Sorry there's not much to go off of here, I can keep testing things if it helps any.

**Comment by DB628** [comment]
FYI, Mine also was never resolved, still the same loop once playing a video, as mentioned originally all settings are off.  In-case it helps my device is a Samsung S23 Plus

I never followed up, bc I simply learned to deal with it and hit the stop button, LOL

Sorry about that, been really busy with work, cheers, I'll keep an eye out on this thread if anything changes, I'll test when able

Don

**Comment by deckerst** [comment]
@DB628 could you please paste here the system info you get in the app About > Bug Report section? And do you use a custom ROM?

**Comment by DB628** [comment]
No I don't use a custom ROM, Give me a bit, I'm driving now to work

**Comment by DB628** [comment]
I C&P while stuck at the train crossing, I would of attached the file rather then C&P here, but couldn't find where to attach while waiting here, so hope this helps

Don 👌

**Comment by deckerst** [comment]
Thanks, but I meant the system info (not the logs).

**Comment by DB628** [comment]
Sorry was reading on the fly, see below, hope this helps 😉


Package: deckers.thibault.aves
Installer: com.android.vending
Aves version: 1.13.12-play, build 163
Flutter: stable 3.27.4
Android version: 16, API 36
Android build: BP2A.250605.031.A3.S916U1UES6EYK5
Device: samsung SM-S916U1
Display: pixel ratio=2.8125, logical=384.0x832.0, physical=1080.0x2340.0
Support: dynamic colors=true, geocoder=true, HDR=false, wide gamut=false
Mobile services: ready
Connectivity: wifi
System locales: en_US
Storage volumes: /storage/emulated/0/
Storage grants: /storage/emulated/0/Movies/, /storage/emulated/0/Pictures/, /storage/emulated/0/DCIM/, /storage/emulated/0/Android/media/
Error reporting: false
